build/__main__.py

```diff
@@ -96,6 +101,28 @@ def main(cli_args, prog=None):  # type: (List[str], Optional[str]) -> None
     args = parser.parse_args(cli_args)
 
     distributions = []
+    config_settings = {}
+
+    if args.config_setting:
+        for arg in args.config_setting:
+            if sys.version_info >= (3,):
+                data = arg.split('=', maxsplit=1)
+            else:  # pragma: no cover
```

Comment:
eh, why do not cover this?

Reply:
Because then we have to merge the reports, and currently Python 3 and Python 2 jobs run in different jobs. I might fix it later, by creating a new job that gathers the report from the tests and pushes them to codecov.

Reply:
I'd prefer to leave it uncovered to indicate job to be done rather than override it like this 👍

Reply:
Hum, looks like code codecov should be able to merge multiple reports, we just might have to pass the `PYTHON` env var. I will open a PR for this.